<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgHeightfieldDimension</class>
 <widget class="QDialog" name="CQDlgHeightfieldDimension">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>146</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Heightfield Dimensions</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>X-size [m]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqSizeX">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-size [m]</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqSizeY">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-scaling</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqScalingZ">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqOk">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>75</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
